spec/requests/posts/new_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+describe 'GET /posts/new' do
+  subject { get "/posts/new"; response }
+
+  context "when user is signed in" do
+    let!(:user) { create :user }
+
+    before { login_as user }
+
+    it { is_expected.to be_successful }
+
+    it { is_expected.to render_template 'new' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.